<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerConfView</class>
 <widget class="QWidget" name="PlayerConfView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorButton" name="colorButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Player type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="playerTypeComboBox">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Humanoid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Computer</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="moveDelayLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="moveDelaySpinButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: red; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>PlayerConfView</receiver>
   <slot>_nameEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PlayerConfView</receiver>
   <slot>_playerTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_nameEdited(QString)</slot>
  <slot>_playerTypeChanged(int)</slot>
 </slots>
</ui>
